modules/apps/object/object-api/src/main/java/com/liferay/object/petra/sql/dsl/DynamicObjectDefinitionTableUtil.java

```diff
@@ -43,6 +43,21 @@ public static String getAlterTableAddColumnSQL(
 		return sql;
 	}
 
+	public static String getAlterTableAddColumnSQL(
+		String tableName, String columnName, String dbType,
+		String sqlColumnNull) {
```

Comment:
I agree, but I used that name because it is already being used [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-api/src/main/java/com/liferay/object/petra/sql/dsl/DynamicObjectDefinitionTableUtil.java#L54) and it refers to the same thing